homeassistant/components/isy994/binary_sensor.py

```diff
@@ -425,6 +428,12 @@ async def async_added_to_hass(self) -> None:
         # Start the timer on bootup, so we can change from UNKNOWN to OFF
         self._restart_timer()
 
+        if (last_state := await self.async_get_last_state()) is not None:
+            # Only restore the state if it was previously ON (Low Battery)
+            if last_state.state == STATE_ON:
+                self._computed_state = True
+                self.async_write_ha_state()
```

Comment:
```suggestion
```

I don't think the state write is needed 

https://github.com/home-assistant/core/blob/77dd0b2fd3da885571f4d78f4ab23b6531d2b3c3/homeassistant/helpers/entity.py#L804

Reply:
Fixed.